EIPS/eip-20.md

```diff
@@ -1,52 +1,46 @@
 ---
 eip: 20
 title: Token Standard
+description: A standard interface for tokens
 author: Fabian Vogelsteller <fabian@ethereum.org>, Vitalik Buterin <vitalik.buterin@ethereum.org>
+discussions-to: https://github.com/ethereum/EIPs/issues/20
+status: Final
 type: Standards Track
 category: ERC
-status: Final
 created: 2015-11-19
 ---
 
-## Simple Summary
-
-A standard interface for tokens.
-
-
 ## Abstract
 
-The following standard allows for the implementation of a standard API for tokens within smart contracts.
-This standard provides basic functionality to transfer tokens, as well as allow tokens to be approved so they can be spent by another on-chain third party.
-
+This EIP standardizes an interface for fungible tokens within smart contracts.
+The interface has basic transfer functionality, and allows tokens to be approved so they can be spent by another on-chain third party.
```

Comment:
I believe this change goes beyond the minimum required to fit current standards.

```suggestion
The following standard allows for the implementation of a standard API for tokens within smart contracts.
This standard provides basic functionality to transfer tokens, as well as allow tokens to be approved so they can be spent by another on-chain third party.
```